docker/README.md

```diff
@@ -47,25 +47,36 @@ The CMD launches Huginn via the scripts/init script. This may become the ENTRYPO
 
 Simple stand-alone usage:
 
-    docker run -it -p 5000:5000 cantino/huginn
+    docker run -it -p 3000:3000 cantino/huginn
 
 To link to another mysql container, for example:
 
-    docker run --rm --name newcentury_mysql -p 3306 \
+    docker run --rm --name mysql \
```

Comment:
How about naming the container `mysql_huginn` or something like it to avoid conflicts with other mysql containers?